indicators/NO_GJEN_002/R/NO_GJEN_002.qmd

```diff
@@ -0,0 +1,1108 @@
+names(lidar_df) <- c("x", "y", "lidar_height")
+
+p1 <- ggplot() +
+  geom_raster(data = rgb_df, aes(x = x, y = y, fill = rgb(red, green, blue, maxColorValue = 255))) +
+  scale_fill_identity() +
+  coord_fixed() +  
+  theme_void() +
+  ggtitle("RGB Image 2020")
+
+p2 <- ggplot() +
+  geom_raster(data = pred_df, aes(x = x, y = y, fill = pred_height)) +
+  scale_fill_viridis_c(option = "turbo", limits = c(0, 14), name = "Height (m)") +
+  coord_fixed() +  
+  theme_void() +theme(legend.position = "none")+
+  ggtitle("Predicted Height 2020")
+
+
+p3 <- ggplot() +
+  geom_raster(data = lidar_df, aes(x = x, y = y, fill = lidar_height)) +
+  scale_fill_viridis_c(option = "turbo", limits = c(-0.5, 14), name = "Height (m)") +
+  coord_fixed() +  
+  theme_void() +
+  ggtitle("LiDAR Height 2017")
+
+combined_plot <- (p1 + p2 + p3) + plot_layout(ncol = 3)
+
+print(combined_plot)
+
+```
+
+```{r}
+img <- image_read("/data/P-Prosjekter2/412421_okologisk_tilstand_2024/Andrew/meta/github/ecRxiv_encroachment/indicators/NO_GJEN_002/img/tileComparison.png")
+
+plot(img)
+```
+
+Initial validation attempts showed poor performance relating to airborne images taken outside of the months of June, July and August. This was expected, since the model was trained on vegetation at peak foliation. For this reason, imagery collected outside of this month range was excluded from validation. We also filtered out points that intersected with the FKB building footprints layer, points that were in shadow (calculated by sampling the blue channel of the input RGB images and filtering out based on a threshold value). The resulting csv file after filtering contains c. 30000 points with LiDAR and Meta model based canopy heights. Future iterations of this script will automate these vaildation steps. 
+
+```{r}
+val <- read.csv('/data/P-Prosjekter2/412421_okologisk_tilstand_2024/Andrew/meta/github/ecRxiv_encroachment/indicators/NO_GJEN_002/data/NiNpointSamples.csv')
+ggplot(val, aes(x = lidmax, y = metamax)) +
+  geom_point(size=2.5, alpha=0.2, stroke=0, color = 'darkgray') +  # Scatter plot
+  geom_smooth(method = "lm", se = FALSE, color = "darkblue") +  # Line of best fit
+  theme_minimal() +  # Clean theme
+  coord_fixed(ratio = 1) +  # Square aspect ratio
+  labs(title = "LiDAR vs meta modeled canopy heights", x = "LiDAR (m)", y = "Meta model (m)") +  # Labels
+  # Add R-squared value to the plot
+  annotate("text", x = Inf, y = Inf, label = paste("R² =", round(summary(lm(metamax ~ lidmax, data = val))$r.squared, 2)), 
+           hjust = 1.1, vjust = 1.1, size = 5, color = "darkblue")+ xlim(0,30) + ylim(0,30) 
+```
+
+### 10.2 Comparison of modelled vs LiDAR gjengroing index
+
+We can plot a scatterplot to compare LiDAR based index with the Meta based index. Note that in the subsample of NiN areas used here, most of them have a good index values. Future work to scale this over a greater number of sites should give a better spread of data for a more robust comparison. 
+```{r}
```

Comment:
Split so there is one figure output per block. Otherwise the cross referencing doesnt work.